src/main/java/com/ericsson/ei/controller/EntryPointConstantsUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.ericsson.ei.controller;
+
+public final class EntryPointConstantsUtils {
+
+    public static final String AGGREGATED_OBJECTS = "/aggregated-objects";
+
+    private EntryPointConstantsUtils() {
+        throw new AssertionError();
```

Comment:
Is there a reason why there is an AssertionError in the constructor of this static class? The constructor is already private and cannot be used.

Reply:
It is not very important But I like it as a good prog practice.
"this prevents even the native class from calling this ctor as well"... REF: http://www.javapractices.com/topic/TopicAction.do?Id=2